page-info.php

```diff
@@ -0,0 +1,171 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="info">
+    <div class="container">
+      <div class="grid-row">
+
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+
+    $team = get_post_meta($post->ID, '_igv_teammate_group', true);
+    $collaborators = get_post_meta($post->ID, '_igv_collaborator_group', true);
+    $clients = get_post_meta($post->ID, '_igv_client_group', true);
+    $services = get_post_meta($post->ID, '_igv_service_group', true);
+    $contact = get_post_meta($post->ID, '_igv_contact', true);
+?>
+
+        <article <?php post_class('grid-item item-s-12'); ?> id="post-<?php the_ID(); ?>">
+            <h2>About</h2>
+    <?php the_content(); ?>
+
+    <?php
+    if (!empty($team)) {
+    ?>
+
+          <section>
```

Comment:
each section should be a row and if needed the inner content split into grid items